opencog/reasoning/pln/rules/member-to-evaluation.scm

```diff
@@ -0,0 +1,58 @@
+; ============================================================================= 
+; GeneralMemberToEvaluationRule
+;	MemberLink( B (SatisfyingSetLink( Variable $X 
+;		(EvaluationLink (pred D (ListLink X C))))))
+;		becomes
+;	EvaluationLink (pred D (ListLink B C))
+; -----------------------------------------------------------------------------
```

Comment:
Same remark as for GeneralEvaluationToMemberRule, it can be decomposed into a combination of simpler rules.